### PR TITLE
changes for v1 enterprise and token-info commands for cli

### PR DESCRIPTION
**Pull request by ammar2168**
- made changes to enterprise show and update commands corresponding to enterprise v1 API
- added command token show 
- added test case for command token show
- altered test case for command enterprise show and update

### CODE REVIEW

File: esper/controllers/enterprise/enterprise.py [hunk 1]
```diff
@@ -29,42 +29,40 @@ def _enterprise_basic_response(self, enterprise, format=OutputFormat.TABULATED):
...
-                'name': enterprise.name,
-                'display_name': enterprise.display_name,
-                'registered_name': registered_name,
-                'address': address,
-                'location': location,
-                'zipcode': zipcode,
-                'email': email,
-                'contact_person': contact_person,
-                'contact_number': contact_number
+                'Enterprise Id': enterprise.id,
```
Comment (abdulMuneerM) [inline esper/controllers/enterprise/enterprise.py]: Don't change the field name? Any specific reason?

File: esper/controllers/token/token.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+from esper.ext.db_wrapper import DBWrapper
+from esper.ext.utils import validate_creds_exists, parse_error_message
+
+
+class Token(Controller):
+    class Meta:
+        label = 'token'
+
+        # text displayed at the top of --help output
+        description = 'token command'
```
Comment (tarun91vas) [inline esper/controllers/token/token.py]: Can we have a little detailed description ?

File: esper/controllers/token/token.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+        # text displayed at the bottom of --help output
+        epilog = 'Usage: espercli token'
```
Comment (tarun91vas) [inline esper/controllers/token/token.py]: update readme also.

File: esper/controllers/enterprise/enterprise.py (hunk 1 above)
Comment (ammar2168) [reply esper/controllers/enterprise/enterprise.py]: @jerynmathew suggested to change.

Comment (ammar2168) [reply esper/controllers/enterprise/enterprise.py]: so that It doesn't look like code variables

**Pull request closed without merging**